docs: add engagement rules (#34)
PyTorchLightning/ecosystem-ci: README.md

   ```
1. Finally, create a draft PR to the repo!

### Additional suggestions and engagement rules

> - To qualify for GPU machines we require your project to have 100+ GitHub stars (please note that this is for capacity reasons and may change in future)

Reviewer: ```suggestion
- To qualify for GPU machines we require your project to have 100+ GitHub stars (please note that this is for capacity reasons and may change in the future)
```

> - For qualification to use GPU machine we set min number of GitHub starts as 100+ (please note it s for capacity reasons and we this threshold may change in the future :)

Reviewer 2: ```suggestion
- For qualification to use GPU machine we set minimum number of GitHub starts as 100+ (please note its for capacity reasons and this threshold may change in the future :] )
```

Author: used https://github.com/PyTorchLightning/ecosystem-ci/pull/34#discussion_r796040348

> - For qualification to use GPU machine we set min number of GitHub starts as 100+ (please note it s for capacity reasons and we this threshold may change in the future :)

Reviewer 3: ```suggestion
- To qualify for GPU machines we require your project to have 100+ GitHub stars (please note that this is for capacity reasons and may change in future)
```

> - For qualification to use GPU machine we set min number of GitHub starts as 100+ (please note it s for capacity reasons and we this threshold may change in the future :)
> - (**Optional**) Join our [Slack](https://join.slack.com/t/pytorch-lightning/shared_invite/zt-12iz3cds1-uyyyBYJLiaL2bqVmMN7n~A) channel `#alerts-ecosystem-ci` to be notified if your project is breaking
> - (**Kind request**) include Lightning badge in your readme:
>   ```md
>   [![Lightning](https://img.shields.io/badge/-Lightning-792ee5?logo=pytorchlightning&logoColor=white)](https://pytorchlightning.ai)
>   ```

Reviewer: ```suggestion
- For the qualification to use GPU machine we set min number of GitHub stars as 100+ (please note it is for capacity reasons and this threshold may change in the future :)
- (**Optional**) Join our [Slack](https://join.slack.com/t/pytorch-lightning/shared_invite/zt-12iz3cds1-uyyyBYJLiaL2bqVmMN7n~A) channel `#alerts-ecosystem-ci` to be notified if your project is breaking
- (**Kind request**) Include [Lightning badge](https://img.shields.io/badge/-Lightning-792ee5?logo=pytorchlightning&logoColor=white) in your readme:
  ```md
  [![Lightning](https://img.shields.io/badge/-Lightning-792ee5?logo=pytorchlightning&logoColor=white)](https://pytorchlightning.ai)
  ```
```

Author: not sure what is the change and seems GH marked it as outdated, could you please check it again...

Reviewer: some typos, can't remember. Grammarly suggested some... looks like they are resolved now
